d2l-button-shared-styles.html

```diff
@@ -1,6 +1,12 @@
 <link rel="import" href="../polymer/polymer.html">
 <style is="custom-style">
 	html {
+		--d2l-button-clear-focus: {
+			box-shadow: 0 0 0 4px rgba(0, 0, 0, 0);
+		};
```

Comment:
It looks fine from button/demo page within this repo. But from our LMS, the button has a mark left from the box-shadow looks like this 
![image](https://cloud.githubusercontent.com/assets/5272785/24162445/ed332710-0e3d-11e7-85c7-0783af5a1540.png)